doc/Manuals/ig-ss_x-road_v6_security_server_installation_guide.md

```diff
@@ -245,6 +246,9 @@ To configure support for hardware security tokens (smartcard, USB token, Hardwar
 
         sudo service xroad-signer restart
 
+With some HSM drivers, it is not possible to configure specific slots for the server to use.
+To prevent scanning a lot of unnecessary slots specific slot indexes can be configured in `/etc/xroad/conf.d/signer.ini`
```

Comment:
signer.ini should not be modified by end user. /etc/xroad/conf.d/local.ini should be used instead